katas/helloworld/jorge/jorge.csproj

```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Hi @Luisjor , please remove this file since the HelloWorld katas have been updated to do not be standalone projects